pkg/profilecreator/profilecreator.go

```diff
@@ -259,7 +259,7 @@ func (ghwHandler GHWHandler) GetReservedAndIsolatedCPUs(reservedCPUCount int, sp
 		return "", "", fmt.Errorf("can't obtain CPU info from GHW snapshot: %v", err)
 	}
 	if reservedCPUCount < 0 || reservedCPUCount > int(cpuInfo.TotalThreads) {
```

Comment:
is  reservedCPUCount == int(cpuInfo.TotalThreads)  a valid choice? Not a single cpu for workloads? We may want to error out or to issue a warning at least